vehicles/src/modules/permit-application-payment/permit/permit.controller.ts

```diff
@@ -118,10 +119,10 @@ export class PermitController {
    * @returns The id of new voided/revoked permit a in response object {@link ResultDto}
    *
    */
-  @Roles({
-    userAuthGroup: IDIR_USER_AUTH_GROUP_LIST,
-    oneOf: [Role.VOID_PERMIT],
-  })
+  @Permissions({
+    userRole: IDIR_USER_ROLE_LIST,
+    oneOf: [Claim.VOID_PERMIT],
+  } as IRole)
```

Comment:
I dont think you need to typecast it as IRole

Reply:
<img width="854" alt="image" src="https://github.com/user-attachments/assets/6a932eec-c58c-4c70-b658-e4ad0ce90965">
Without explicit casting, there's an eslint error.